pkg/download/download.go

```diff
@@ -49,7 +49,7 @@ func (p *protocol) List(ctx context.Context, mod string) ([]string, error) {
 func (p *protocol) Info(ctx context.Context, mod, ver string) ([]byte, error) {
 	const op errors.Op = "protocol.Info"
 	v, err := p.s.Get(ctx, mod, ver)
-	if storage.IsNotFoundError(err) {
+	if errors.ErrNotFound(err) {
```

Comment:
can we, as this is check, stick with IsErrNotFound?

Reply:
@michalpristas how about `IsNotFoundErr` ? to stay consist with std lib's `os.IsNotExist` -- grammatically speaking.

Reply:
👍 Great!